README.md

```diff
@@ -1,148 +1,75 @@
-        </tr>
-        <tr>
-            <td align="left">siteIdentifier</td>
-            <td align="left">For the Bold BI Enterprise edition, it should be like `site/site1`. For Bold BI Cloud, it should be an empty string.</td>
-        </tr>
-        <tr>
-            <td align="left">authorizationUrl</td>
-            <td align="left">Url of the 'GetDetails' action in the ASP.NET application.</td>
-        </tr>
-        <tr>
-            <td align="left">userEmail</td>
-            <td align="left">UserEmail of the Admin in your Bold BI, which would be used to get the dashboard list.</td>
-        </tr>
-        <tr>
-            <td align="left">userPassword</td>
-            <td align="left">Password of the Admin in your Bold BI, which would be used to get the dashboard list.</td>
-        </tr>
-    </tbody>
-    </table>
-
-
-### Install npm
-
-To install all dependent packages, use the below command
-
-```bash
-npm install
-```
-
-### Install Bold BI Embedded SDK package
-
-To install the Bold BI Embedded SDK package using the following command,
-
-```bash
-npm install -save @boldbi/boldbi-embedded-sdk
-```
-
-### Run/Serve
-
-To run the samples, use the below command
-
-```bash
-npm start
-```
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
-
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
-
-### `npm test`
-
-```bash
-npm test
-```
-
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
-
-```bash
-npm run build
-```
-
-Please refer to the [help documentation](https://help.boldbi.com/embedded-bi/javascript-based/samples/v3.3.40-or-later/react-core/#how-to-run-the-sample) to know how to run the sample.
-
-## Online Demos
+  3. Build your .NET project by executing the `dotnet build` command in the terminal.
+
+  4. To install all dependent packages, use the following command `npm install`.
+  
+  5. To run the application, use the command `dotnet run` in the terminal. After executing the command, the application will automatically launch in the default browser. You can access it at the specified port number (e.g., https://localhost:5001/).
+    
+      ![dashboard image](https://github.com/boldbi/aspnet-core-sample/assets/91586758/4af68f49-ffc0-400a-a323-55a3f3600a1d)
+
+Please refer to the [help documentation](https://help.boldbi.com/embedding-options/embedding-sdk/samples/react-core/#how-to-run-the-samples) to know how to run the sample.
+
+## Important notes
+
+It is recommended to not store passwords and sensitive information in configuration files for security reasons, in a real-world application. Instead, you should consider using a secure application, such as Key Vault, to safeguard your credentials.
```

Comment:
It is recommended not to store passwords and sensitive information in configuration files for security reasons in a real-world application. Instead, you should consider using a secure application, such as Key Vault, to safeguard your credentials.

Reply:
Content updated